.github/workflows/linkchecker.yaml

```diff
@@ -0,0 +1,26 @@
+name: Check links
+
+on:
+  schedule:
+    - cron: "10 7 * * *"
+
+jobs:
+  run-cypress:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v2
+
+      - name: Build image
+        run: DOCKER_BUILDKIT=1 docker build --tag ubuntu-com .
+
+      - name: Run linkchecker against the server
+        run: |
+          docker run -d -p 80:80 ubuntu-com
```

Comment:
I think you need to add here `DATABASE_URL=sqlite:///db.sqlite3 ubuntu-com `

Reply:
nope. it will read `.env`, which is part of the test

Reply:
I don't think it reads from .env here in the docker build step